jpo-security-svcs/src/main/java/us/dot/its/jpo/sec/controllers/SignatureController.java

```diff
@@ -15,7 +15,7 @@
  ******************************************************************************/
 package us.dot.its.jpo.sec.controllers;
 
-import joptsimple.internal.Strings;
+
```

Comment:
Note: The joptsimple.internal.Strings package wasn't functioning as expected after updating the Kafka version. This has been replaced with the StringUtils package.